utils/phenotype_loading.py

```diff
@@ -641,14 +641,59 @@ def parse_first_occurrence(x):
     return mt
 
 
-def load_covid_data(all_samples_ht: hl.Table, covid_data_path: str, wave: str = '01'):
+def load_covid_data(all_samples_ht: hl.Table, covid_data_path: str, hes_main_path: str,hes_diag_path: str, death_path: str, wave: str = '01'):
     print(f'Loading COVID wave {wave}...')
+
+    #### PATH OF DATA USED IN TESTING ####
+    # def get_covid_data_path(wave: str = '20201103'):
+    #     return f'gs://ukb31063/ukb31063.covid19_test_result.{wave}.txt'
+
+    # def get_hes_main_data_path(wave: str = '20200909'):
+    #     return f'gs://ukb31063/ukb31063.hesin.{wave}.txt'
+
+    # def get_hes_diag_data_path(wave: str = '20200909'):
+    #     return f'gs://ukb31063/ukb31063.hesin_diag.{wave}.txt'
+
+    # def get_death_data_path(wave: str = '20201012'):
+    #     return f'gs://ukb31063/ukb31063.death.{wave}.txt'
     covid_ht = hl.import_table(covid_data_path, delimiter='\t', missing='', impute=True, key='eid')
+    hes_main_ht = hl.import_table(hes_main_path, delimiter='\t', missing='', impute=True, key='eid')
+    hes_diag_ht = hl.import_table(hes_diag_path, delimiter='\t', missing='', impute=True, key='eid')
+    death_ht = hl.import_table(hes_death_path, delimiter='\t', missing='', impute=True, key='eid')
+    
+    ## Death Register Data Cleaning
+    death_ht = death_ht.annotate(death_date = hl.experimental.strptime(death_ht.date_of_death+ ' 00:00:00', '%d/%m/%Y %H:%M:%S', 'GMT'))
+    ## HES Inpatient Data Cleaning
+    hes_main_ht = hes_main_ht.annotate(diagdate = hl.or_else(hes_main_ht.epistart,hes_main_ht.admidate))
+    hes_main_ht = hes_main_ht.annotate(diag_date = hl.experimental.strptime(hes_main_ht.diagdate+ ' 00:00:00', '%d/%m/%Y %H:%M:%S', 'GMT'))
+    hes_main_ht = hes_main_ht.annotate(admi_date = hl.experimental.strptime(hes_main_ht.admidate+ ' 00:00:00', '%d/%m/%Y %H:%M:%S', 'GMT'))
+    
+    hes_ht = hes_main_ht.key_by("eid","ins_index").join(hes_diag_ht.key_by("eid","ins_index"))#Join HES Inpatient Tables
+    
+    hes_pcr_pos = hes_ht.filter(hes_ht.diag_icd10=="U071").key_by("eid") # Subset Patients with Positive PCR Test Results
+    hes_other = hes_ht.filter((hes_ht.diag_icd10!="U071")&(hes_ht.diag_icd10!="U072")).key_by("eid")# Subset Patients with diagnoses not related to PCR Test
```

Comment:
Just to be a stickler about Python style: spaces around all `&`, `==`, etc, one space after each `,` and two spaces after the line of code before `#` if you're putting in a comment (and then one space after the `#`).